docs/user/dashboard/dashboard.asciidoc

```diff
@@ -0,0 +1,472 @@
+[[dashboard]]
+= Dashboard
+
+[partintro]
+--
+
+A _dashboard_ is a collection of panels that you use to analyze your data. On a dashboard, you can add a variety of panels that 
+you can rearrange and tell a story about your data. Panels contain everything you need, including visualizations, 
+interactive controls, Markdown, and more.  
+
+With *Dashboard*, you can:
+
+* Add multiple panels to see many aspects and views of your data in one place.
+
+* Arrange panels for analysis and comparison.
+
+* Add text and images to provide context to the panels and make them easy to consume.
+
+* Create and apply filters to focus on the data you want to display.
+
+* Control who can use your data, and share the dashboard with a small or large audience. 
+
+* Generate reports based on your findings.
+
+To begin, open the menu, go to *Dashboard*, then click *Create dashboard*.
+
+[role="screenshot"]
+image:images/Dashboard_example.png[Example dashboard]
+
+[float]
+[[dashboard-read-only-access]]
+=== [xpack]#Read only access#
+If you see
+the read-only icon in the application header,
+then you don't have sufficient privileges to create and save dashboards. The buttons to create and edit
+dashboards are not visible. For more information, see <<xpack-security-authorization>>.
+
+[role="screenshot"]
+image::images/dashboard-read-only-badge.png[Example of Dashboard read only access indicator in Kibana header]
+
+[float]
+[[types-of-panels]]
+== Types of panels
+
+Panels contain everything you need to tell a story about you data, including visualizations, 
+interactive controls, Markdown, and more.
+
+[cols="50, 50"]
+|===
```

Comment:
I'm a bit confused by the way that this is talking about "types of panels" but it's mostly listing types of visualizations. Do we really need to explain what an area chart is? Can we remove this entire section?

Reply:
I've received a lot of positive feedback about this section. At this point, I don't think it would be a good idea to remove this.